Dockerfile

```diff
@@ -13,13 +17,13 @@ RUN apk --no-cache add shadow su-exec docker
 RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
 
 #Install kubectl
-RUN curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.10.2/bin/linux/amd64/kubectl && chmod +x /usr/bin/kubectl
+RUN curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl && chmod +x /usr/bin/kubectl
 
 #Install aws cli and azure cli
-RUN apk --no-cache add su-exec docker groff python py-pip gettext procps xz && \
+RUN apk --no-cache add su-exec docker groff python py-pip gettext procps xz jq linux-headers=4.4.6-r2 && \
```

Comment:
I think we should move `linux-headers` to the `build` virtual package

Reply:
moved